app/src/main/res/values-v28/arrays.xml

```diff
@@ -39,6 +39,24 @@
         <item>always</item>
     </array>
 
+    <array name="message_expiring_descriptions">
+        <item>@string/nc_expire_message_off</item>
+        <item>@string/nc_expire_message_four_weeks</item>
+        <item>@string/nc_expire_message_one_week</item>
+        <item>@string/nc_expire_message_one_day</item>
+        <item>@string/nc_expire_message_eight_hours</item>
+        <item>@string/nc_expire_message_one_hour</item>
+    </array>
+
+    <array name="message_expiring_values">
+        <item>expire_0</item>
+        <item>expire_2419200</item>
+        <item>expire_604800</item>
+        <item>expire_86400</item>
+        <item>expire_28800</item>
+        <item>expire_3600</item>
+    </array>
```

Comment:
these values won't be available on any pre v28 Android version and I guess crash, so you need to put them into `/values/` I guess if that file is available, else create it. Sidenote not for this PR, arrays.xml seems to have a lot of duplicated for "all" and "v28+", so some housekeeping to do in the future I think.

Reply:
that's true. deleted them here and tested with v27 device. everything works.